libnmstate/netapplier.py

```diff
@@ -149,6 +149,7 @@ def _apply_ifaces_state(
                 )
             if not nmclient.can_disable_ipv6():
                 _disable_ipv6(desired_state)
+            nmclient.drop_client()
```

Comment:
At this point the Checkpoint teardown is not done. Isn't this a problem?

Reply:
The checkpoint should has its own mainloop and nmclient in case the main process goes wrong.